platform/registry.go

```diff
@@ -80,7 +79,7 @@ var platformList = []blockatlas.Platform{
 	&algorand.Platform{},
 	&nano.Platform{},
 	&harmony.Platform{},
-	&polkadot.Platform{CoinIndex: coin.KSM},
+	//&polkadot.Platform{CoinIndex: coin.KSM},
```

Comment:
Can you enable polkadot? but disable in integration tests?

Reply:
Will be enabled by @Pantani PR when will be fixed